honeybee_vtk/legend_parameter.py

```diff
@@ -314,6 +318,22 @@ def colorset(self, val) -> None:
                 f'A ColorSet objects expected. Instead got {val}.'
             )
 
+    @property
+    def reverse_colorset(self) -> bool:
+        """A boolean to specify whether the colorset should be reversed."""
+        return self._reverse_colorset
+
+    @reverse_colorset.setter
+    def reverse_colorset(self, val) -> None:
+        if not val:
+            self._reverse_colorset = False
+        elif isinstance(val, bool):
+            self._reverse_colorset = val
```

Comment:
When you have a boolean property where `False` and `None` mean the same thing, the following setter is good enough:
```
@reverse_colorset.setter
    def reverse_colorset(self, val) -> None:
        self._reverse_colorset = bool(val)
```